### PR TITLE
Mention MIGRATION_MODULES for Django 1.7 users in integration docs.

### PR DESCRIPTION
**Pull request by zerok**
If you don't have the correct `MIGRATION_MODULES` set before running migrate you end up with similar error messages as have been described in https://github.com/divio/django-cms/issues/3436.

### CODE REVIEW

**Comment by yakky** [comment]
LGTM
@evildmp, any comments?

**Comment by yakky** [comment]
Fix #3741

File: docs/how_to/integrate.rst [hunk 1]
```diff
@@ -495,6 +495,27 @@ setting should look like::
     }
 
 
+Given django CMS' support for Django 1.6.x, Django 1.7 (or later) users have to
```
Comment (evildmp) [inline docs/how_to/integrate.rst]: "django CMS's support"

**Comment by zerok** [comment]
@evildmp ah, I always thought that if the word already ends with an s there is no additional s supposed to be after the apostrophe. I've looked it up and you are absolutely correct as the additional s would be pronounced in this case. Hurray for learning something :-D Corrected it and spashed the commit.

**Comment by yakky** [comment]
thanks @zerok for the lightning fix!

**Pull request merged**